5calls/app/build.gradle

```diff
@@ -108,3 +108,7 @@ dependencies {
 
 apply plugin: 'com.google.gms.google-services'
 apply plugin: 'org.jetbrains.kotlin.android'
+
+kotlin {
```

Comment:
I really didn't want to modify this file but I needed this change to get tests working on my machine, is there a better way 🤔 ?

Reply:
I'm OK to change gradle to make tests work, although I'm a bit surprised we need it since the new tests are in Java.